src/optimizer/core/src/transform.rs

```diff
@@ -373,6 +373,18 @@ fn create_inline_qhook(url: JsWord, symbol: &str) -> ast::CallExpr {
                     kind: ast::StrKind::Synthesized,
                 }))),
             },
+            // ast::ExprOrSpread {
```

Comment:
Just remove?

Reply:
yes! it was a stupid idea to include import.meta.url